src/gc/unix/gcenv.unix.cpp

```diff
@@ -471,7 +486,7 @@ uint64_t GCToOSInterface::GetPhysicalMemoryLimit()
     long pages = sysconf(_SC_PHYS_PAGES);
     if (pages == -1) 
     {
-        return 0;
+        return 0;SC_PAGE_SIZE
```

Comment:
@jkotas Yeah.  My editor went haywire writing random clipboard strings throughout files.  I corrected a few dozen, but this one must have slipped through.